orc/src/main/java/org/apache/iceberg/orc/ORC.java

```diff
@@ -789,7 +808,210 @@ static Reader newFileReader(InputFile file, Configuration config) {
     ReaderOptions readerOptions = OrcFile.readerOptions(config).useUTCTimestamp(true);
     if (file instanceof HadoopInputFile) {
       readerOptions.filesystem(((HadoopInputFile) file).getFileSystem());
+    } else {
```

Comment:
The `newFileReader(String, ReaderOptions)` above should be deprecated, right?

Reply:
I moved the OrcFile.createReader method into the new method and deprecated the previous one. That is a package local method, if acceptable I can remove it as it has no other uses.

Reply:
Yeah, rather than deprecating we should just remove it if it isn't public.

Reply:
deleted